src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +39,16 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" to={"/" + (query ? `?q=${query}#${searchInputId}` : "")}>
```

Comment:
Nah, don't do anything if this means adding a configuration variable. I was more thinking of using relative paths like `..` if the relative path can be reliably computed. Defining a root path as a config variable always ends up being a PITA.